tasks.py

```diff
@@ -22,7 +22,6 @@
 # so we build it here rather so that we don't clutter up the tasks.
 locale_abstraction_instructions = " ".join([
     "makemessages",
-    "-l de -l es -l fr -l nl -l pl -l pt_BR",
```

Comment:
This is not 100% related, it’s just we don’t need to explicitly to do that anymore since files are stored in an external repo